aggregator/aggregator.py

```diff
@@ -0,0 +1,73 @@
+# Manages aggregated logs
+# It stores the logs in a sqlite database
+
+import argparse
+import requests
+import datetime
+import time
+import urllib
+
+# Project imports
+import log_manager
+
+# Constants
+SECONDS_PER_DAY = 86400
+SERVER_LIST_FILE = '../servers.txt'
+
+# Globals
+log_mgr = log_manager.LogManager()
+
+def retrieve_server_list():
+  with open(SERVER_LIST_FILE, 'rb') as server_file:
+    return server_file.read().splitlines() 
+
+def date_to_timestamp(date):
+  return int(time.mktime(datetime.datetime.strptime(date, "%Y-%m-%d").timetuple()))
+
+def timestamp_to_date(timestamp):
+  return datetime.datetime.utcfromtimestamp(timestamp).strftime('%Y-%m-%d')
+
+def request_log_from_server(server, date):
+  print "Retrieving logs from server <http://" + server + "> for date: " + date + "..."
+  url = 'http://%s/logs?%s' % (server, urllib.urlencode({ 'date': date }))
+  print url
+  r = requests.get(url)
+  if r.status_code == 200:
+    return r.text
```

Comment:
Is this right? I'm not sure if you have to open a stream to receive the content of the log file or you can just do this.